internal/rvinfo/rvinfo.go

```diff
@@ -116,11 +116,19 @@ func UpdateRvInfo(rvInfo *[][]protocol.RvInstruction, index int, rvMap map[proto
 		newRvInfo[index] = append(newRvInfo[index], protocol.RvInstruction{Variable: protocol.RVDns, Value: utils.MustMarshal(rvMap[protocol.RVDns].(string))})
 	}
 
-	host := rvMap[protocol.RVIPAddress].(string)
```

Comment:
This copies the fix that is proposed here: https://github.com/fido-device-onboard/go-fdo-server/pull/30

Tests fail without it.